content/applications/essentials/in_app_purchase.rst

```diff
@@ -1,57 +1,186 @@
+   (phone) SMS` icon appears to the right.
+
+   Following this, click the :guilabel:`📱 (phone) SMS` icon, and a :guilabel:`Send SMS Text
+   Message` pop-up window appears.
+
+   Finally, type a message in the :guilabel:`Message` field of the :guilabel:`Send SMS Text Message`
+   pop-up window. Then, click the :guilabel:`Send SMS` button. Odoo then sends the message, via SMS,
+   to the contact, and logs what was sent in the *chatter* of the contact's form.
+
+   Upon sending the SMS message, the prepaid credits for the *SMS* |IAP| service are automatically
+   deducted from the existing credits. If there are not enough credits to send the message, Odoo
+   prompts the user to purchase more.
+
+.. seealso::
+   For more information on how to use various |IAP| services, and for more in-depth instructions
+   related to SMS functionality in Odoo, review the documentation below:
+
+   - :doc:`Lead mining </applications/sales/crm/acquire_leads/lead_mining>`
+   - :doc:`Enrich your contacts base with Partner Autocomplete
+     </applications/sales/crm/optimize/partner_autocomplete>`
+   - :doc:`SMS essentials </applications/marketing/sms_marketing/essentials/sms_essentials>`
+
+.. _in_app_purchase/credits:
+
+IAP credits
+===========
+
+Every time an |IAP| service is used, the prepaid credits for that service are spent. Odoo prompts
+the purchase of more credits when there are not enough credits left to continue using a service.
+Email alerts can also be set up for when :ref:`credits are low <in_app_purchase/low-credits>`.
+
+Credits are purchased in *Packs* from the `Odoo IAP Catalog
+<https://iap.odoo.com/iap/all-in-app-services>`_, and pricing is specific to each service.
+
+.. example::
+   The `SMS service <https://iap.odoo.com/iap/in-app-services/1>`_ has four packs available, in
+   denominations of:
+
+   - :guilabel:`Starter Pack`: 10 credits
+   - :guilabel:`Standard Pack`: 100 credits
+   - :guilabel:`Advanced Pack`: 500 credits
+   - :guilabel:`Expert Pack`: 1,000 credits
+
+   .. image:: in_app_purchase/packs.png
+      :align: center
+      :alt: Four different packs of credits for the SMS IAP service.
+
+   The number of credits consumed depends on the length of the SMS and the country of destination.
+
+   For more information, refer to the :doc:`SMS Pricing and FAQ
+   </applications/marketing/sms_marketing/pricing/pricing_and_faq>` documentation.
+
+.. _iap/buying_credits:
+
+Buy credits
+-----------
+
+If there are not enough credits to perform a task, the database automatically prompts the purchase
+of more credits.
+
+Users can check the current balance of credits for each service, and manually purchase more credits,
+by navigating to the :menuselection:`Settings app --> Contact --> Odoo IAP --> View My Services`.
+
+Doing so reveals an :guilabel:`IAP Account` page, listing the various |IAP| services in the
+database. From here, click an |IAP| service to open its :guilabel:`Account Information` page, where
```

Comment:
In 15 it looks like it's either "Odoo IAP" or "My Services" - but I don't have any credits to see this in these runbots, so I can't say what it looks lif eif ther are credits purchased. 

![image](https://github.com/odoo/documentation/assets/121518652/83a63195-3567-4f67-9985-247735224950)